dindex_builder/dindex_builder.py

```diff
@@ -2,22 +2,36 @@
 import os
 import time
 import numpy as np
+import pandas as pd
 
 from typing import Dict
 from datasketch import MinHash, MinHashLSH
 
 from dindex_store.discovery_index import DiscoveryIndex
 from dindex_store.common import EdgeType
 
+CSV_DELIMITER = ','
+
+
+def load_data(path, file_type, discovery_graph: DiscoveryIndex):
+    # Populate the nodes table with files under the specified path
+
+    if file_type != 'json' and file_type != 'csv':
+        print(f'File type {file_type} currently not supported')
+        return
 
-def load_json_data(path, discovery_graph: DiscoveryIndex):
-    # Populate the nodes table with JSON files under the specified path
     for filename in os.listdir(path):
         filepath = os.path.join(path, filename)
         if os.path.isfile(filepath):
-            with open(filepath) as f:
-                profile = json.load(f)
-                discovery_graph.add_profile(profile)
+            if file_type == 'json':
+                with open(filepath) as f:
+                    profile = json.load(f)
+                    discovery_graph.add_profile(profile)
+            elif file_type == 'csv':
```

Comment:
What is this CSV option? I thought we did not accept any other input than JSON for profiles right now.